### DISCUSSION TITLE
Welcome to Camell-web3 Discussions!

### DISCUSSION THREAD

**Discussion by Camell-Cloud** [Camell-Cloud/Camell-web3#6] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jeonilshin** [comment, marked as answer]
잘 부탁 드립니다!
